src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,30 @@
+package com.example;
+
+/**
+ * A customer is a person who visits a shop for buying goods.
+ *
+ * <p>The customer can have a loyalty card that has a specific number.
+ */
+public final class Customer {
+    /**
+     * In order to use previously collected bonuses, the customer uses the card during a purchase.
+     *
+     * <p>The absence of a card does not prevent the customer from purchasing the goods.
+     */
+    private final LoyaltyCard card;
+
+    /**
+     * Loyalty card is assigned to a specific customer.
+     *
+     * <p>The card provides customer with bonuses that can be used to pay for purchases.
+     *
+     * <p>The customer's card number is unchanged.
+     *
+     * <p>The card number must be checked.
+     *
+     * @param card is an additional customer identifier.
```

Comment:
Please remove the dot at the end of the parameter description.

Also, please mention that a customer may not have a card. Refer to my comment on nullability of the parameter.